### PR TITLE
Adding .plist file example

### PR DESCRIPTION
**Pull request by Jasdev**

### CODE REVIEW

File: cheat_sheet.swift [hunk 1]
```diff
@@ -167,36 +167,36 @@ func addUserToAddressBook(firstName: String, lastName: String, jobTitle: String,
...
+
+
                 }
-                
+
                 success = ABRecordSetValue(newContact, kABPersonNoteProperty, "added via iPhone App", &error)
                 success = ABAddressBookAddRecord(addressBook, newContact, &error)
                 success = ABAddressBookSave(addressBook, &error)
-                
+
```
Comment (zackshapiro) [inline cheat_sheet.swift]: @Jasdev what's with all of these lines?

File: readme.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ These are things that I lookup frequently or solutions I spent some time on. I h
 * Add a person to AddressBook
 * Use HexColors codes for UIColor
 * Setting and retrieving NSUserDefaults
+* Reading a .plist file
 * More to come
 
 Created by [@ZackShapiro](http://twitter.com/zackshapiro)
```
Comment (zackshapiro) [inline readme.md]: Add your name on this line too :)

File: cheat_sheet.swift (hunk 1 above)
Comment (Jasdev) [inline cheat_sheet.swift]: Atom, likes removing leading spaces on empty line :smile: 

Want me to revert these?

Comment (zackshapiro) [inline cheat_sheet.swift]: Ah, of course. Just a difference between XCode and Atom. I wonder if
there's an XCode add on to remove whitespace

**Comment by zackshapiro** [comment]
@Jasdev whitespace – http://stackoverflow.com/a/11830067/602210

**Comment by Jasdev** [comment]
@zackshapiro nice! Added :+1:

**Comment by zackshapiro** [comment]
@Jasdev will you rebase please? then i'll merge

**Comment by Jasdev** [comment]
@zackshapiro done!

**Pull request merged**